application/controllers/admin/DataEntry.php

```diff
@@ -357,8 +357,7 @@ public function import($surveyid)
             }
             $imported = 0;
             $sourceResponses = new CDataProviderIterator(new CActiveDataProvider($sourceTable), 500);
-            /* @var boolean preserveIDs */
-            $preserveIDs = boolval(App()->getRequest()->getPost('preserveIDs'));
+            $preserveIDs = (bool)App()->getRequest()->getPost('preserveIDs');
```

Comment:
Why remove the comment ?
(else no reason to replace boolval by cast as bool)

Reply:
I have put the comment back